### ISSUE TITLE
Poker planning meeting

### ISSUE THREAD

**Issue by NepheriusIsTaken** [SOEN341-Group1/KoalaCal#90]
@g-harel @RemusCM @ekowpo @jaslatendresse @eCom0327 @RanjuShetty @faisalhabibgit 

The teacher said that we need to do a poker planning meeting before the next sprint to decide on whatever we want to do. Technically, we should have done it last friday. 

To have a planning meeting that is slightly more fruitful than the last ones we did, me and Gabriel reviewed and revamped our user stories to make them a little more specific (I think we had mostly epics until now). Some stories might still be too large in scope. You can find those new user stories in the user story folder.

I created user story cards to be used in the poker planning meeting. They are in an excel file in the branch "DOCU_WIP_UserStoryCards". As we didn't have the time to do it and I'll be pretty busy tomorrow, I still need someone to go through the different user stories and add a relevant acceptance test to them. See requirements slides p. 44 and p. 66 for more details as to what an acceptance test is.

Just as a FYI, here are the things we are evaluated on for the next sprint:
https://docs.google.com/spreadsheets/d/1ysOms7WuOkHhe9R71aMVlW-LZl9kuwrT4Trg77s48FE/edit#gid=0

**Comment by jaslatendresse** [comment]
Yes we really need to meet weekly and have productive meetings but it's been difficult to reach out to everyone. We should all agree one particular day and everyone should be there. For now we need to settle on exactly where we're going so we can start building. And yes, I found the scope a bit big in some stories. We don't have that much time.

**Comment by faisalhabibgit** [comment]
From my side, I can meet you guys in person or online since I live beside school, as long as you guys plan something that comes in the time frame I mentioned as my availability under issue **Weekly meetings schedule discussion**
>

**Comment by RemusCM** [comment]
I feel like we don't really have a lot of common availabilities. If the problem persists, we might not actually need the whole team for meetings, as anyways, we need to do minutes of the meetings. We could split into two times maybe?

**Comment by jaslatendresse** [comment]
I do agree that we can do everything over discord or slack or any online platform, but like @faisalhabibgit said, we need to meet at least once in person, the whole team ASAP to settle things, give story points, assign tasks to people and then we can start building and meeting online as needed. As it is difficult to have the participation of everyone on github, we can discuss that tomorrow before the meeting with the TA as it is not a sprint and everyone will be there and we won't be graded.

**Comment by faisalhabibgit** [comment]
I want us to get 5/5 on Sprint-2. :smiley: (and this would be possible if we meet all the requirement mentioned in the file that @NepheriusIsTaken added up in this conversation)
Also I am going to ask him for the bonus mentioned on the lecture slides for Travis using maven if we can get back out one point that we lost last sprint :disappointed:

**Comment by faisalhabibgit** [comment]
<style="color:red"> test </style>
